File: myoTest.py
#Worked with Marc, Nicholas, and Shyam @ HackGTeeny

from Graphics import *
from Myro import *
from time import *

win = Window("Controller", 500, 500)
init('sim')

def handleKeyRelease(win, event):
    if event.key == "Up":
        forward(1)
    if event.key == "Down":
        backward(1)
    if event.key == "Left":
        turnLeft(1)
    if event.key == "Right":
        turnRight(1)
    if event.key == "s":
        stop()


onKeyPress(handleKeyRelease)

File: myoTest.lua
--Worked with Marc, Nicholas, and Shyam @ HackGTeeny

scriptId = 'com.thalmic.scripts.presentation'

function forward()
    myo.debug("forward")
	myo.keyboard("down_arrow", "press")
end

function backward()
	myo.debug("backward")
    myo.keyboard("up_arrow", "press")
end

function upDown()
	if pose == "waveIn" then
		forward()
	elseif pose == "waveOut" then
		backward()
	end
end
